docker.d/pre-stop.sh

```diff
@@ -30,13 +30,11 @@ POLL_INTERVAL=5
 
 started=$(date +%s)
 
-echo "Sending SIGUSR1 signal to process id $SCRIPTWORKER_PID"
 
-kill -s USR1 $SCRIPTWORKER_PID
 
 while true; do
-    echo "checking status..."
-    kill -0 $SCRIPTWORKER_PID
+    echo "Sending SIGUSR1 signal to process id $SCRIPTWORKER_PID"
```

Comment:
nit:
```suggestion
    echo "Sending SIGUSR1 to process id $SCRIPTWORKER_PID"
```

Also, I'm wondering how useful it is to be repeating that line every 5 seconds?

Reply:
True, I left the line because it was spamming before anyway but I guess there's not much value in it. The pre-stop hook logs are all emitted at the same time when it fails anyway (and the other logs continue to get output at the same time) so everything is confusing anyway.

Reply:
Moved it out of the loop and removed the line that spammed about "waiting 5s" too